.gitignore

```diff
@@ -13,3 +13,4 @@ htmlcov
 venv/
 .mypy_cache/
 *.egg-info/
+.vscode/
```

Comment:
Is should not exclude `tasks.json` as this repository provides that one.

```suggestion

# Visual Studio Code
.vscode/*
!.vscode/tasks.json
```

Reply:
The `settings.json` was the trigger for me, can also only exclude that.